casts/rows/src/UIForeground/UIForeground.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import type { UIForegroundType } from './types';
+import { PrimaryPart, SecondaryPart } from './Parts';
+import { ActionElement, IconElement, TextElement, NumberElement } from './Elements';
+import { Container } from './Container';
+
+export function SectionElement() {
+    return <View />;
+}
+
+export const UIForeground: UIForegroundType = {
+    Container,
+
+    PrimaryPart,
+    SecondaryPart,
+
+    ActionElement,
+    IconElement,
+    NumberElement,
+    SectionElement,
+    TextElement,
```

Comment:
Too verbose if you ask me, you can omit `Element` without loosing meaning (btw it contradicts with React terminology, where `element` means a component presented in JSX, aka the one that `React.createElement` returns)

Reply:
`Element` changed to `Cell`